client/test/utils/specs_dispatcher.js

```diff
@@ -0,0 +1,16 @@
+var TestDispatcher = function TestDispatcher() {
```

Comment:
May be the mock can just use client/lib/flux/dispatcher/dispatcher.coffee which is a self-contained reusable class and would be closer to appDispatcher behaviour when we want to test more complex behaviour such  as waitFor.